test/evergreen/find_gperftools.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+# Currently evergreen machine doesn't contain gperftools binaries. The aim of this script is to
+# create a caching mechanism for fetching gperftools version 2.9.1. First we check if the binaries
+# for the machine are present in 10gen S3 bucket. If not, we download the sources, build the
+# binaries and upload the binaries to 10gen S3 Bucket.
+#
+# Note: The specific version number of gperftools 2.9.1 follows what mongodb currently uses. 
+#
+set +o verbose
+set +o errexit
+
+# This script requires the aws key, aws secret, build variant, and boolean describing if we are
+# compiling for a cmake build. Example usage: ./find_gerftools.sh aws_key aws_secret ubuntu2004 true
+if [ "$#" -ne 4 ]; then
+    echo "Illegal number of parameters."
+    exit 1
+fi
+
+# Export the aws key and aws secret which are necessary to use aws cli.
+export AWS_ACCESS_KEY_ID=$1
+export AWS_SECRET_ACCESS_KEY=$2
+build_variant=$3
+is_cmake_build=$4
+
+# Track the original directory, so we can build in the directory that calls this script.
+dir=$(pwd)
+
+# Fetch the gperftools library through either building the sources of gerftools 2.9.1 or directly
+# downloading the binaries from the S3 bucket.
+fetch_gperftools()
+{
+    echo "-- FETCH GPERFTOOLS --"
+    aws s3 ls "s3://build_external/jiechenbo_build/tcmalloc_${build_variant}.tgz"
```

Comment:
Do you intend to also remove/rename `jiechenbo_build` when you get the new s3 bucket credentials? Maybe `tcmalloc_build`

Reply:
Yep! will be changing this, once we have the S3 bucket credentials. Thanks!